### DISCUSSION TITLE
【RSS】生成的内容没有包含代码块

### DISCUSSION THREAD

**Discussion by lainbo** [tangly1024/NotionNext#2001] [Q&A]
**描述bug**
在生成的RSS链接中, 没有包含文章中代码块内的内容, 官方模板也存在这个问题

**复现步骤**
1. 打开[官方模板中的文章,](https://preview.tangly1024.com/article/example-1) 查看文章内容, 在文章中**存在**代码块内容
2. 打开[官方模板的RSS链接](https://preview.tangly1024.com/rss/feed.xml), 检查RSS内容, **不存在**代码块中的代码内容

**期望的正常结果**
期望在生成的RSS链接的内容中, 包含每篇文章中完整的内容

**截图**
![image](https://github.com/tangly1024/NotionNext/assets/46062895/c3cd0915-206c-48d2-9aaf-55062c8fcf1b)

**环境**

- 【必填】NotionNext版本 4.2.0
- 【必填】我自己使用gitbook没有生成代码块, 官方模板用的hexo也没有生成
- 【可选】操作系统: Windows 11
- 【可选】浏览器 Chrome 121

**补充说明**
无

**Comment by xxxuuu** [comment]
刚刚升级 `react-notion-x` 到最新版，然后这里的 ssr 改为 `true` 就可以了
https://github.com/tangly1024/NotionNext/blob/5a985b3095d558a04688c0c85ce8ad2e4e6ba506/components/NotionPage.js#L194-L201
